server/src/server.js

```diff
@@ -290,11 +290,17 @@ app.post("/event", function (req, res) {
   }
   hashUserId(req.deviceId).then((userUuid) => {
     let userAnalytics = ua(config.gaId, userUuid.toString());
-    userAnalytics.event(
-      bodyObj.event,
-      bodyObj.action,
-      bodyObj.label
-    ).send();
+    let params = Object.assign(
+      {},
+      bodyObj.options || {},
```

Comment:
I was curious about this, so I looked it up: Object.assign does not throw an exception for null or undefined source values, so you don't actually need the `|| {}` here.